oracle/src/oracleimporter.cpp

```diff
@@ -289,8 +293,9 @@ int OracleImporter::importTextSpoiler(CardSet *set, const QVariant &data)
         }
 
         colors.removeDuplicates();
-        relatedCards = QStringList();
-        reverseRelatedCards = QStringList();
+        //Fortunately, there are no split cards that flip, transform or meld.
```

Comment:
I think we'll need an overhaul of this section if that ever happens, not some quick fix.